src/svcutilcore/src/Microsoft/Tools/ServiceModel/SvcUtil/OptionsStatics.cs

```diff
@@ -80,65 +80,24 @@ internal static class Targets
 
```

Comment:
Looks like the Cmd class and Abbr class needs some clean up as well

Reply:
fixed